README.md

```diff
@@ -111,6 +111,42 @@ $ cabal run --flag interop Interop.Pipes
 * [Interop.Vector](https://github.com/composewell/streamly-examples/blob/master/examples/Interop/Vector.hs): Converting streamly
   stream type to and from `vector` stream type.
 
+## Comparing Haskell Performance with C and Rust
```

Comment:
Keeping language-specific files in a distinguished dir might be less intimidating.
- Haskell files in `haskell/` folder.
- Rust files in `rust/` folder.

Reply:
later, currently we have very few examples in other languages.